numpy/_core/function_base.py

```diff
@@ -533,6 +533,8 @@ def add_newdoc(place, obj, doc, warn_on_python=True):
     """
     new = getattr(__import__(place, globals(), {}, [obj]), obj)
     if isinstance(doc, str):
+        if "${ARRAY_FUNCTION_LIKE}" in doc:
+            doc = overrides.get_array_function_like_doc(new, doc)
```

Comment:
Not sure I love the implicit nature.  I think I would prefer a kwarg for this.  But I suppose that may be a matter of opinion.

Reply:
happy to go with your prefered style. What would you suggest we name the argument ?

Reply:
Just go with something that seems sensible to you, I promise not to bike-shed ;).

Reply:
just to be clear though, did you mean a kwarg on `overrides.get_array_function_like_doc` of `add_newdoc` ? (I assume the former ?)